crates/store/re_chunk_store/src/writes.rs

```diff
@@ -382,8 +382,10 @@ impl ChunkStore {
                 && old_typ != list_array.value_type()
             {
                 re_log::warn_once!(
-                    "Component column '{}' changed type from {old_typ:?} to {:?}",
+                    "Component '{}' with component type '{}' in entity '{}' changed type from {old_typ:?} to {:?}",
```

Comment:
Pre-existing problem but:

Use `re_arrow_util::format_data_type` when formatting arrow `DataType`!  NEVER `Debug` formatting (it is extremely verbose).

In the next arrow version we can start using `Display` formatting, but never `Debug`